config/jobs/cert-manager/website/cert-manager-website-presubmits.yaml

```
@@ -15,7 +15,7 @@ presubmits:
       preset-service-account: "true"
     spec:
       containers:
-      - image: "eu.gcr.io/jetstack-build-infra-images/golang-nodejs:20220302-b57c609-1.17"
+      - image: "eu.gcr.io/jetstack-build-infra-images/golang-nodejs:latest-1.19"
```

suggestion: I've seen some squeamishness about using any kind of "latest" within Jetstack, and I think I tend to agree with it. Do you think we should use the tag instead?

(From `crane ls eu.gcr.io/jetstack-build-infra-images/golang-nodejs`)

```suggestion
      - image: "eu.gcr.io/jetstack-build-infra-images/golang-nodejs:20221017-bcd2286-1.19"
```

---

Ah, thanks! Did not see that tag in GCR. Updated!